src/store/sdp/index.js

```diff
@@ -7,6 +7,7 @@ const state = {
     isAvailable: false,
     loading: false,
     title: '',
+    thumbnail: [],
```

Comment:
thumbnail should be a single photo
right?

Reply:
since the gallery component only takes array